Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.xaml

```diff
@@ -0,0 +1,51 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    <Style TargetType="controls:ImageCropper">
+        <Setter Property="IsTabStop" Value="False" />
+        <Setter Property="HorizontalAlignment" Value="Stretch" />
+        <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+        <Setter Property="VerticalAlignment" Value="Stretch" />
+        <Setter Property="VerticalContentAlignment" Value="Stretch" />
+        <Setter Property="Background" Value="{ThemeResource SystemControlBackgroundAltHighBrush}" />
+        <Setter Property="Mask" Value="{ThemeResource SystemControlBackgroundAltMediumHighBrush}" />
+        <Setter Property="Padding" Value="80" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:ImageCropper">
+                    <Border Background="{TemplateBinding Background}"
+                            BorderBrush="{TemplateBinding BorderBrush}"
+                            BorderThickness="{TemplateBinding BorderThickness}">
+                        <Grid x:Name="PART_LayoutGrid"
+                              Padding="{TemplateBinding Padding}"
+                              HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+                              VerticalAlignment="{TemplateBinding VerticalAlignment}">
+                            <Canvas x:Name="PART_ImageCanvas">
+                                <Image x:Name="PART_SourceImage"
+                                       Source="{TemplateBinding Source}" />
+                                <Path x:Name="PART_MaskAreaPath"
+                                      Fill="{TemplateBinding Mask}" />
+                                <controls:ImageCropperThumb x:Name="PART_TopThumb"
```

Comment:
Might be best to add these in code and provide options on how they should be displayed

Reply:
This gives the user the opportunity to customize each Thumb style, but you can't do that in code unless you add a lot of ThumbStyle properties.